main/ui/src/main/resources/i18n/strings.properties

```diff
@@ -45,8 +45,10 @@ addvaultwizard.new.locationPrompt=…
 addvaultwizard.new.directoryPickerLabel=Custom Location
 addvaultwizard.new.directoryPickerButton=Choose…
 addvaultwizard.new.directoryPickerTitle=Select Directory
-addvaultwizard.new.fileAlreadyExists=Vault can not be created at this path because some object already exists.
-addvaultwizard.new.locationDoesNotExist=Vault can not be created at this path because at least one path component does not exist.
+addvaultwizard.new.fileAlreadyExists=A file or directory with the vault name already exists.
+addvaultwizard.new.locationDoesNotExist=A directory in the specified path does not exists or cannot be accessed.
```

Comment:
```suggestion
addvaultwizard.new.locationDoesNotExist=A directory in the specified path does not exist or cannot be accessed.
```

Reply:
Should those Strings end with a `.` ?

Reply:
You are right, let's remove the `.`, similar to the "choose password screen".

Reply:
Fixed in 1048ab0ef2d2946eb77f1e981a8b49ae223ec68d